pages/linux/xterm.md

```diff
@@ -0,0 +1,23 @@
+# xterm
+
+> A terminal emulator for the X Window System.
+
+- Open the terminal with a title of `Example`:
+
+`xterm -T {{Example}}`
+
+- Open the terminal in fullscreen mode:
+
+`xterm -fullscreen`
+
+- Open the terminal with a dark blue background and yellow foreground (font color):
+
+`xterm -bg {{darkblue}} -fg {{yellow}}`
+
+- Open the terminal with 100 characters per line and 35 lines in screen position x=200 pixels and y=20 pixels:
```

Comment:
Please add a comma after "35 lines" to make the sentence a little easier to read. And maybe specify screen position as "x=200px, y=20px", for compactness. 😊